core/src/scene/sceneLoader.cpp

```diff
@@ -415,6 +448,33 @@ Filter* SceneLoader::generateNoneFilter(YAML::Node _filter) {
     return (new None(std::move(filters)));
 }
 
+void SceneLoader::parseStyleProps(Node styleProps, StyleParamMap& paramMap, const std::string& propPrefix) {
+
+    for(auto propItr = styleProps.begin(); propItr != styleProps.end(); ++propItr) {
+        Node prop = propItr->first;
+        std::string paramKey;
+        if(propPrefix.length() > 0) {
+            paramKey = propPrefix + ":" + prop.as<std::string>();
+        } else {
+            paramKey = prop.as<std::string>();
+        }
+        if(propItr->second.IsScalar()) {
+            paramMap.emplace(paramKey, propItr->second.as<std::string>());
+        } else if(propItr->second.IsSequence()) {
+            if(prop.as<std::string>() == "color") {
```

Comment:
Yes, its parsing for color.. 

parseColorSequence ... brings [R, G, B]/[R, G, B, A] in cssColorParser recognizable format, which is rgba(r, g, b, a) (all values other than alpha in 0-255 range and alpha in 0.0-1.0 range).